pkg/pillar/hypervisor/xen.go

```diff
@@ -133,12 +133,15 @@ func (ctx xenContext) Setup(status types.DomainStatus, config types.DomainConfig
 	if err = spec.AddLoader("/containers/services/xen-tools"); err != nil {
 		return logError("failed to add xen hypervisor loader to domain %s: %v", status.DomainName, err)
 	}
-
-	// finally we can start it up
 	spec.Get().Process.Args = []string{"/etc/xen/scripts/xen-start", status.DomainName, file.Name()}
 	if config.MetaDataType == types.MetaDataOpenStack {
 		spec.Get().Process.Args = append(spec.Get().Process.Args, "smbios_product")
 	}
+
+	// Grant full access to devices
+	spec.GrantFullAccess()
```

Comment:
```suggestion
	spec.GrantFullAccessToDevices()
```
makes the code easier readable so that no documentation is needed ;-)

(but also feel free to ignore and click Resolve)

Reply:
@rene, do you want to address this one? Or are we good to merge?

Reply:
ok. done.